delta/service/src/test/resources/akka-persistence-test.conf

```diff
@@ -1,22 +1,4 @@
 akka {
-  actor {
-    provider = "akka.cluster.ClusterActorRefProvider"
-    serialize-messages = on
```

Comment:
leave this enabled in the test scope, to ensure we don't forget serialization bindings for types exchanged over the network